pyClient/Makefile

```diff
@@ -34,3 +34,7 @@ syntax: ${PROTOBUF_OUTPUT}
 
 test: ${PROTOBUF_OUTPUT}
 	python -m unittest
+
+test_contracts: ${PROTOBUF_OUTPUT}
+	python test/test_contract_base_mixer.py
```

Comment:
I tried to keep the non-unit tests (those that are commands and require the ethereum RPC host) in `test_commands`.  Could these two be moved into `test_commands` so they cannot every cause any confusion with the unit tests (which `unittest` scans).